packages/cadl-vscode/package.json

```diff
@@ -109,7 +109,7 @@
     "@rollup/plugin-node-resolve": "~13.1.3",
     "@types/mkdirp": "~1.0.1",
     "@types/mocha": "~9.1.0",
-    "@types/node": "~14.0.27",
+    "@types/node": "~17.0.23",
```

Comment:
Shouldn't we keep this at the minimum version we support? Does 16.0.x work? Could we wait for the Node 14 support removal to go in to adress the issue here?

Reply:
Yeah I guess I just took the latest version, should probably work with node 16